webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -0,0 +1,3058 @@
+            self.dframeobs["north"].max(),
+        ]
+
+        self.set_callbacks(app)
+
+    @property
+    def tour_steps(self) -> List[dict]:
+        return [
+            {
+                "id": self.uuid("obsdata-ens_name"),
+                "content": ("Select ensemble to view."),
+            },
+            {
+                "id": self.uuid("obsdata-regions"),
+                "content": ("Region filter. "),
+            },
+            {
+                "id": self.uuid("obsdata-noise_filter"),
+                "content": ("Noise filter. In steps of half of the lowest obs error."),
+            },
+            {
+                "id": self.uuid("obsdata-showerror"),
+                "content": ("Toggle observation error on or off."),
+            },
+            {
+                "id": self.uuid("obsdata-showhistogram"),
+                "content": ("Toggle observation data histogram on or off."),
+            },
+            {
+                "id": self.uuid("obsdata-resetindex"),
+                "content": (
+                    "Use original ordering (as from imported data) or reset index"
+                    + " (can be useful in combination with filters."
+                ),
+            },
+            {
+                "id": self.uuid("obsdata-obsmap_colorby"),
+                "content": ("Select data to use for coloring of the map view plot."),
+            },
+            {
+                "id": self.uuid("obsdata-obsmap_scale_col_range"),
+                "content": (
+                    "Select color range scaling factor used "
+                    + "with the map view plot."
+                ),
+            },
+            {
+                "id": self.uuid("obsdata-graph-raw"),
+                "content": ("Observation data 'raw' plot."),
+            },
+            {
+                "id": self.uuid("obsdata-graph-map"),
+                "content": ("Observation data map view plot."),
+            },
+            {
+                "id": self.uuid("obsdata-info"),
+                "content": (
+                    "Info of the ensembles observation data comparison. "
+                    + "For a direct comparison they should have the same "
+                    + "observation and observation error data."
+                ),
+            },
+            # {
+            #    "id": self.uuid(""),
+            #    "content": ("."),
+            # },
+        ]
+
+    @property
+    def layout(self) -> wcc.Tabs:
+
+        tabs_styles = {"height": "60px", "width": "100%"}
+
+        tab_style = {
+            "borderBottom": "1px solid #d6d6d6",
+            "padding": "6px",
+            "fontWeight": "bold",
+        }
+
+        tab_selected_style = {
+            "borderTop": "1px solid #d6d6d6",
+            "borderBottom": "1px solid #d6d6d6",
+            "backgroundColor": "#007079",
+            "color": "white",
+            "padding": "6px",
+        }
+
+        return wcc.Tabs(
```

Comment:
You could consider splitting this return statement into smalle utility functions, e.g. 

```python
return wcc.Tabs(..., children=[self._seismic_obs_data_layout(), ...])
```
which also will help making the lines shorter (less indentation).